frontend/components/panels/WalletDepositFlowCheckPanel.vue

```diff
@@ -116,10 +116,7 @@ export default Vue.extend({
             return this.desiredAmount && !this.desiredAmount.isNaN();
         },
         isEnoughDeposited(): boolean {
-            if (!this.walletVatAmount || !this.isDesiredAmountValid || this.desiredAmount.isLessThan(0)) {
-                return false;
-            }
-            return this.desiredAmount.isLessThan(this.walletVatAmount);
+            return !this.minimumDepositAmount?.isGreaterThan(0);
```

Comment:
In case `this.minimumDepositAmount` is null, `!this.minimumDepositAmount?.isGreaterThan(0)` will evaluate into `true`. Is it expected? Otherwise I would suggest to use `.isZero()` without negation to avoid this edge case